scss/cayman.scss

```diff
@@ -44,19 +59,19 @@ a {
 .btn {
   display: inline-block;
   margin-bottom: 1rem;
-  color: rgba(255, 255, 255, 0.7);
-  background-color: rgba(255, 255, 255, 0.08);
-  border-color: rgba(255, 255, 255, 0.2);
+  color: rgba($button-text-color, 0.7);
+  background-color: rgba($light-color, 0.08);
+  border-color: rgba($light-color, 0.2);
```

Comment:
For all of these `rgba($light-color, *)` uses, we probably need to come up with variables for each case (e.g. `$btn-bg-color`, `$btn-border-color`, etc.)